Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -110,6 +110,20 @@ void MemoryWidget::CreateWidgets()
   search_layout->addWidget(m_result_label);
   search_layout->setSpacing(1);
 
+  // Address Space
+  auto* address_space_group = new QGroupBox(tr("Address Space"));
+  auto* address_space_layout = new QVBoxLayout;
+  address_space_group->setLayout(address_space_layout);
+
+  m_address_space_effective = new QRadioButton(tr("Effective"));
+  m_address_space_auxiliary = new QRadioButton(tr("Auxiliary"));
+  m_address_space_physical = new QRadioButton(tr("Physical"));
```

Comment:
Should these have localization hints to prevent mistranslation? (I can a case where a translator may be confused by "Physical", at least).